ports/gtl/portfile.cmake

```
@@ -0,0 +1,21 @@
+#header-only library
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO greg7mdp/gtl
+    REF 1.00
+    SHA512 842a5e2634919a04fdd87995a8ada2f949c51a070a7c175043e0f9105a93248325023f85b28f9406276c2912a0fb4015a2e9ba30113d4a0214492da0dc5e5716
+    HEAD_REF main
+)
+
+# Use greg7mdp/gtl's own build process, skipping examples and tests
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)
+vcpkg_cmake_install()
+
+# Delete redundant directories
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib ${CURRENT_PACKAGES_DIR}/debug ${CURRENT_PACKAGES_DIR}/share/doc)
```

```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib" "${CURRENT_PACKAGES_DIR}/debug" "${CURRENT_PACKAGES_DIR}/share/doc")
```

Please add quotes here...